build_config/webpack.devserver.conf.js

```diff
@@ -10,7 +10,7 @@ var webpack = require('webpack');
 //
 
 var port = 9002;
-var addr = 'media-atom-maker-assets.local.dev-gutools.co.uk';
+var addr = 'video-assets.local.dev-gutools.co.uk';
```

Comment:
I'm fairly sure I've seen this go through before. I think the branch probably needs to be updated with master.